### ISSUE TITLE
Lots of GetNumX() functions lack return type

### ISSUE THREAD

**Issue by emmericp** [Ketho/vscode-wow-api#124]
https://github.com/Ketho/vscode-wow-api/blob/98df833d58aa86ef96b06e85755e10880a8476aa/EmmyLua/API/GlobalAPI/API3.lua#L603

These sound like they return number

**Comment by Ketho** [comment]
Yeah it's a lot of functions that are still undocumented on the wiki, before they can get parsed to annotations ☹️
